fixtures/services/bat-safety.yaml

```diff
@@ -1,18 +1,21 @@
 slug: bat-safety
 
-topic: healthcare-and-prevention
-
+topic: healthcare-prevention
+image: curbside-compost
 contact: 311
 
 title_en: Keep yourself safe around bats in Austin
+title_es: Keep yourself safe around bats in Austin
+title_vi: Keep yourself safe around bats in Austin
+title_ar: Keep yourself safe around bats in Austin
 
-steps_en:
-<h2>Follow these steps to keep yourself safe if you come into contact with a bat</h2>
+steps_en: |
```

Comment:
@LauraTrujillo  In order to do fancy html in the body below, we need this pipe operator ,`|`. This tells the YAML file to expect a multiline string.